scheduler.rst

```diff
@@ -0,0 +1,129 @@
+Scheduler
+=========
+
+.. versionadded:: 6.3
+
+    The Scheduler component was introduced in Symfony 6.3 and is marked
+    as experimental.
+
+The Scheduler component provides a way to schedule periodical tasks or handling
+recurring messages without using an external tool.
+
+The component shares somes keys concepts with :ref:`Messenger <messenger>`.
+
+Installation
+------------
+
+You can install the Scheduler component with:
+
+.. code-block:: terminal
+
+    $ composer require symfony/scheduler
+
+.. include:: /components/require_autoload.rst.inc
+
+.. _register-schedule-provider:
+
+Register a Schedule provider
+----------------------------
+
+A Schedule provider is a class defining your schedule : which messages should 
+be processed depending a trigger you choose::
+
+    namespace App\Schedule;
+
+    use App\Message\EndofTrialMessage;
+    use Symfony\Component\Scheduler\Attribute\AsSchedule;
+    use Symfony\Component\Scheduler\RecurringMessage;
+    use Symfony\Component\Scheduler\Schedule;
+    use Symfony\Component\Scheduler\ScheduleProviderInterface;
+
+    #[AsSchedule('trial')]
```

Comment:
```suggestion
    #[AsSchedule('default')]
```

I'm thinking we should just use default here. Multiple schedules is an advanced use-case imo.